packager.gemspec

```diff
@@ -22,9 +22,9 @@ Gem::Specification.new do |s|
 
   s.required_ruby_version = '>= 1.9.3'
 
-  s.add_dependency 'dsl_maker', '~> 0.1', '>= 0.1.1'
-  s.add_dependency 'fpm', '~> 1.4', '>= 1.1.0'
-  s.add_dependency 'thor', '~> 0.0', '>= 0.19.0'
+  s.add_dependency 'dsl_maker', '>= 0.1.1'
```

Comment:
This needs to be `'~> 1.0', '>= 1.0.0'` in order to force the usage of the version with the MIT license.

Reply:
Should this be changed to MIT, its currently GPL2?